source/class/cv/util/ScriptLoader.js

```diff
@@ -256,7 +256,9 @@ qx.Class.define('cv.util.ScriptLoader', {
         loader.removeListener('loaded', this._onLoaded, this);
         loader.removeListener('failed', this._onFailed, this);
       });
-      loader.start();
+      loader.start(); // NOTE: Actually we'd need a `.catch()` here when the
+      // loading of a plugin is failing. But then no error message at all is
+      // shown. Why?!? FIXME!
       return loader;
```

Comment:
The error is sent as notification for topic `cv.loading.error` but no one is listening on this topic.

When you add the `catch(e => {})` here and:

```js
        'cv.loading.error': {
          type: 'error',
          icon: 'message_attention'
        },
```

here https://github.com/CometVisu/CometVisu/blob/develop/source/class/cv/ui/PopupHandler.js#L58 the error is shown as popup.

I guest we should add that to this PR because currently it does not fix anything.